Lib/test/test_platform.py

```diff
@@ -328,6 +328,27 @@ def test_java_ver(self):
 
     def test_win32_ver(self):
         res = platform.win32_ver()
+        self.assertEqual(len(res), 4)
+        if support.MS_WINDOWS:
+            release, version, csd, ptype = res
+            if release:
+                releases = []
+                releases.extend(r for _k, r in platform._WIN32_CLIENT_RELEASES)
+                releases.extend(r for _k, r in platform._WIN32_SERVER_RELEASES)
+                self.assertIn(release, releases)
+            if version:
+                # It is rather hard to test explicit version without
+                # going deep into the details.
+                self.assertIn('.', version)
```

Comment:
Maybe assert that `int(v) for v in version.split('.')` succeeds?

Reply:
Yes, I guess it is safe to add this check, since we already do the same in `platform.py`: https://github.com/python/cpython/blob/eb947cdc1374842a32fa82249ba3c688abf252dc/Lib/platform.py#L451